cea/datamanagement/databases_verification.py

```diff
@@ -10,7 +10,8 @@
 COLUMNS_DISTRICT_GEOMETRY = ['Name', 'height_ag', 'floors_ag']
 COLUMNS_ZONE_AGE = ['built', 'roof', 'windows', 'partitions', 'basement', 'HVAC', 'envelope']
 COLUMNS_ZONE_OCCUPANCY = ['MULTI_RES', 'OFFICE', 'RETAIL', 'SCHOOL', 'HOTEL', 'GYM', 'HOSPITAL', 'INDUSTRIAL',
-                          'RESTAURANT','SINGLE_RES', 'SERVERROOM', 'SWIMMING', 'FOODSTORE', 'LIBRARY', 'COOLROOM', 'PARKING']
+                          'RESTAURANT','SINGLE_RES', 'SERVERROOM', 'SWIMMING', 'FOODSTORE', 'LIBRARY', 'COOLROOM',
+                          'PARKING', 'LAB', 'MUSEUM', 'LIBRARY', 'UNIVERSITY']
```

Comment:
Didn't know there was a 'UNIVERSITY' archetype in SG (in CH we have just used 'SCHOOL' for lecture halls and specific archetypes for any other uses). Are these schedules taken from ASHRAE?

Just curious, no action needed.

Reply:
I think most of the schedules for SG are copied from CH.